### PR TITLE
[RM chapter 5] Language corrections.

### PR DESCRIPTION
**Pull request by mpgreaves**

### CODE REVIEW

**Review by walterkozlowski — APPROVED** [review]

**Comment by walterkozlowski** [comment]
@bfcohen, can you please review these language corrections, which I approved? You are our English and editing expert in residence!

**Comment by bfcohen** [comment]
Have to log in to the correct account.

File: doc/ref_model/chapters/chapter05.rst
Comment (bfcohen100) [inline doc/ref_model/chapters/chapter05.rst:1]: Needs some more edits. I have to reopen the file to do the edits.

**Review by bfcohen100 — COMMENTED** [review]
Needs some more editing.  I will complete the review and then reopen to do the edits.  I think that will be easier.

File: doc/ref_model/chapters/chapter05.rst [hunk 1]
```diff
@@ -1,24 +1,25 @@
...
-can be hosted to execute on. Workload Flavours provide a mechanism to specify the VM or Pod sizing information to host
-the workload. Depending on the requirements of the workloads, a VM or a Pod will be deployed as per the specified
-Flavour information on a node configured as per the specified Profile. Not only do the nodes (the hardware) have to be
-configured but some of the capabilities also need to be configured in the software layers (such as Operating System and
-Virtualisation Software). Thus, a Profile can be defined in terms of configuration needed in the software layers, the
-Cloud Infrastructure Software Profile, and the hardware, the Cloud Infrastructure Hardware Profile.
-
-Cloud Infrastructure Software profile description
+Cloud Infrastructure node (host or server). :ref:`chapters/chapter02:profile extensions (specialisations)`
+may specify additional configurations. Workloads utilise profiles to describe the configuration of nodes on which they
```
Comment (bfcohen100) [inline doc/ref_model/chapters/chapter05.rst:6]: replace utilise with use

**Review by bfcohen100 — COMMENTED** [review]
Needs more edits

**Review by bfcohen — COMMENTED** [review]
Overall looks OK.  Needs some polishing.

File: doc/ref_model/chapters/chapter05.rst (hunk 1 above)
Comment (CsatariGergely) [reply doc/ref_model/chapters/chapter05.rst:6]: fixed in https://github.com/anuket-project/RM/pull/17

**Pull request merged**